app/bundles/EmailBundle/Tests/Swiftmailer/SendGrid/Callback/SendGridApiCallbackTest.php

```diff
@@ -24,7 +24,7 @@ public function testSupportedEvents()
             ->disableOriginalConstructor()
             ->getMock();
 
-        $sendGridApiCallback = new SendGridApiCallback($transportCallback);
+        $sendGridApiCallback = new MomentumCallback($transportCallback);
```

Comment:
Sendgrid is suppose to return MomentumCallback?

Reply:
Fixed